lerobot/common/policies/diffusion/configuration_diffusion.py

```diff
@@ -98,7 +98,7 @@ class DiffusionConfig:
 
     # Inputs / output structure.
     n_obs_steps: int = 2
-    horizon: int = 16
+    horizon: int = 10
```

Comment:
I think this change should be reverted right? To keep in line with the default PushT policy.